docs/docs/cmd/tenant/auditlog/auditlog-report.md

```diff
@@ -0,0 +1,46 @@
+# tenant auditlog report
+
+Gets audit logs from the Office 365 Management API
+
+## Usage
+
+```sh
+m365 tenant auditlog report [options]
+```
+
+## Options
+
+`-c, --contentType <contentType>`
+: Audit content type of logs to be retrieved, should be one of the following: AzureActiveDirectory, Exchange, SharePoint, General, DLP.
+
+`-s, --startTime [startTime]`
+: Start time of logs to be retrieved. Start time and end time must both be specified (or both omitted) and must be less than or equal to 24 hours apart.
+
+`-e, --endTime [endTime]`
+: End time of logs to be retrieved. Start time and end time must both be specified (or both omitted) and must be less than or equal to 24 hours apart.
```

Comment:
Please check my comments regarding the endTime omission if you specify a starttime

Reply:
What I am thinking is to give our explanation accordingly. Something around these lines - 

> If you haven't entered the `endTime`, command will automatically consider the `endTime `as 24 hours after the `startTime` and will generate the report accordingly.

As per this [question](https://github.com/pnp/cli-microsoft365/pull/2001#discussion_r551129374), can I give the explanation for the case where they have not entered startTime as well.?

Reply:
Yes